### PR TITLE
feat: added loader, alert

### PR DESCRIPTION
**Pull request by muskaanshah**
Added loader for products page
Alert component created, it hasn't been implemented yet though

### CODE REVIEW

File: src/components/Alert/Alert.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import "./alert.css";
+
+function Alert({ type, msg }) {
```
Comment (shobit1337) [inline src/components/Alert/Alert.jsx:3]: Why not use the arrow function here?

**Review by shobit1337 — APPROVED** [review]
Everything looks cool. Your code is clean and has good namings.

File: src/components/Alert/Alert.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+			class={`alert borderradius-0-5 ${
+				type === "success"
+					? "alert-success"
+					: type === "warning"
+					? "alert-warning"
+					: "alert-error"
+			}`}
+		>
+			<span class="material-icons">
+				{type === "success"
```
Comment (manismk) [inline src/components/Alert/Alert.jsx]: change class to className

**Review by manismk — APPROVED** [review]
Rest looks good. good naming convention

**Pull request merged**